config/config.exs

```diff
@@ -25,3 +25,9 @@ config :phoenix, :json_library, Jason
 # Import environment specific config. This must remain at the bottom
 # of this file so it overrides the configuration defined above.
 import_config "#{Mix.env()}.exs"
+
+#config cors_plug
```

Comment:
Is this comment intentional?

Reply:
yeees...should it be more descriptive? 😅

Reply:
changed it to `#configures cross origin compatability` 👼

Reply:
You probably don't need that comment. This is an Elixir config file... so we can guess that it configures stuff. And also the line below it has the word config :)